client/src/shared/AddOfferInputHandler.tsx

```diff
@@ -0,0 +1,47 @@
+import { useFormik } from "formik";
+import * as Yup from "yup";
+
+export const phoneRegExp =
```

Comment:
What do you think about moving phoneRegExp into some separate file called eg. constants.ts and keep it there because it is used in several files?